### PR TITLE
Created linking table, put all tables in one sql file

### PR DESCRIPTION
**Pull request by Benson-J**

### CODE REVIEW

File: sql/pokedex_hb.sql [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+-- Table structure for table `status`
+--
+
+DROP TABLE IF EXISTS `status`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `status` (
+  `user_id` int(11) unsigned NOT NULL,
+  `pokemon_id` int(11) unsigned NOT NULL,
+  `seen_caught` int(1) unsigned NOT NULL
```
Comment (hoptond) [inline sql/pokedex_hb.sql]: I'm going to presume you intended that this value should either be a 0 or a 1. At the moment, I can put in really large numbers like 1241243349. Furthermore, what does each number mean? Does 0 mean unseen? Does 1 mean caught or seen? From the variable naming and data type it should be immediately obvious what data in a colum represents.

You might want to add a foreign key constraint here too, referring to a status type in a different table.

File: sql/pokedex_hb.sql [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+--
+-- Table structure for table `status`
+--
+
+DROP TABLE IF EXISTS `status`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `status` (
+  `user_id` int(11) unsigned NOT NULL,
```
Comment (hoptond) [inline sql/pokedex_hb.sql:42]: I can refer to users that don't exist. Pls add foreign key constraints

File: sql/pokedex_hb.sql [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  `pokemon_id` int(11) unsigned NOT NULL,
```
Comment (hoptond) [inline sql/pokedex_hb.sql]: See above, but replace users with pokemon

File: sql/pokedex_hb.sql [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `status`
+--
+
+DROP TABLE IF EXISTS `status`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `status` (
```
Comment (hoptond) [inline sql/pokedex_hb.sql:41]: This doesn't have any indexing or unique ids per row. When our users table grows in size, searches will become slow. It's also possible to add duplicate entries, so the status table can say that user 1 hasn't seen, has seen, and has caught pokemon 57 all at the same time.

**Review by hoptond — CHANGES_REQUESTED** [review]

File: sql/pokedex_hb.sql (hunk 1 above)
Comment (Benson-J) [reply sql/pokedex_hb.sql]: Changed data type to bit, allowing for NULL, 0, & 1; for unseen, seen, & caught respectively.  A comment has been added to the field to reflect this.

File: sql/pokedex_hb.sql (hunk 2 above)
Comment (Benson-J) [reply sql/pokedex_hb.sql:42]: Foreign key to table:user field:id created.

File: sql/pokedex_hb.sql (hunk 3 above)
Comment (Benson-J) [reply sql/pokedex_hb.sql]: Foreign key to table:pokemon field:id created.

File: sql/pokedex_hb.sql (hunk 4 above)
Comment (Benson-J) [reply sql/pokedex_hb.sql:41]: Unique pair constraint added for user_id & pokemon_id.

File: sql/pokedex_hb.sql [hunk 5]
```diff
@@ -40,8 +40,12 @@ DROP TABLE IF EXISTS `status`;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `status` (
```
Comment (hoptond) [inline sql/pokedex_hb.sql:41]: Any reason there's no id column to refer to the identity of each entry in this table?

**Review by hoptond — APPROVED** [review]

File: sql/pokedex_hb.sql (hunk 5 above)
Comment (Benson-J) [reply sql/pokedex_hb.sql:41]: No id necessary for linking tables.

**Pull request merged**